test/integration/client-side-encryption/driver.test.ts

```diff
@@ -297,4 +297,111 @@ describe('Client Side Encryption Functional', function () {
       });
     });
   });
+
+  describe(
```

Comment:
Is auto_encrypter.test.ts a better place for this test?

Reply:
In the unit directory? I set this up to be integration because the unit coverage we already have did not catch the issue introduced, I want to know that `find` will return the right object